.github/workflows/build.yml

```diff
@@ -75,6 +75,8 @@ jobs:
     - name: Build
       run: |
         export PATH=~/.npm-global/bin:$PATH
+        # make sure nothing was manually added to the package.json
+        yarn install --frozen-lockfile
```

Comment:
`make install` already calls yarn install... could you please update the makefile instead

Reply:
`yarn install --frozen-lockfile` is something that should only need to be run in ci. It is basically just a check to make sure that a contributor didn't use `npm install` to add a package, or manually add something to the package.json and never ran `yarn install` to install things

Reply:
Adding `yarn install --frozen-lockfile` to `make install` would throw an error if you tried to add something to a package.json and follow up with a `make install` (something that should be okay to do). Without `yarn install --frozen-lockfile` in `make install`, adding something to the package.json and then running `make install`, would just install the packages and update the `yarn.lock`, which is normally the expected behavior

Reply:
The npm equivalent of this is `npm ci`: https://docs.npmjs.com/cli/v7/commands/npm-ci